packages/relay/src/lib/types/spendingPlanConfig.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { SubscriptionType } from '../db/types/hbarLimiter/subscriptionType';
+
+/**
+ * Represents the configuration for a spending plan.
+ * Pre-configured spending plans are specified in a `spendingPlansConfig.json` file in the root of the project.
+ * @interface SpendingPlanConfig
+ */
+export interface SpendingPlanConfig {
+  /**
+   * The name of the spending plan.
+   * @type {string}
+   */
+  name: string;
+
+  /**
+   * ETH addresses associated with the spending plan.
+   * @type {string[]}
+   * @optional
+   */
+  ethAddresses?: string[];
+
+  /**
+   * IP addresses associated with the spending plan.
+   * @type {string[]}
+   * @optional
+   */
+  ipAddresses?: string[];
+
+  /**
+   * The spending plan tier.
+   * @type {SubscriptionType}
+   */
+  subscriptionTier: SubscriptionType;
```

Comment:
Wondering why this is called `subscriptionTier` instead of `subscriptionType` like the other references. The `HbarSpendingPlan` class refers to it as `subscriptionType`, and the enum is named `SubscriptionType`. Other spending repositories also use `SubscriptionType`. I would recommend renaming it to `subscriptionType` for better consistency across the codebase.

Reply:
Done, changed to `subscriptionType`